frontend/src/slicer/orderSlice.ts

```diff
@@ -51,12 +58,12 @@ export const getOneOrder = createAsyncThunk<IResponseGetAllOrders, string, { sta
 export const createOrder = createAsyncThunk<IResponseGetAllOrders, void, { state: RootState, dispatch: Dispatch }>(
     'orders/create',
     async (_, {getState, dispatch}) => {
-        if (!validateOrder(getState())) {
+        if (!validateBeforeSendingToBackend(getState())) {
```

Comment:
why not directly call `validateOrder` here?

Reply:
because I need to reuse validate order outside of the slice where I won't have access to the getState function of the ThunkApi